src/amo/api/users.js

```diff
@@ -35,7 +35,8 @@ export function editUserAccount({ api, picture, userId, ...editableFields }: {|
     const form = new FormData();
     // Add all the editable fields, one by one.
     Object.keys(editableFields).forEach((key: string) => {
-      form.set(key, editableFields[key]);
+      // We cannot send `null` values, only string values.
+      form.set(key, editableFields[key] === null ? '' : editableFields[key]);
```

Comment:
is it worth it to do something like this ~:
```javascript
const fieldValue = editableFields[key];
form.set(key, fieldValue === null ? '' : fieldValue);
```

Reply:
sure!